src/ByteSizeLib/ByteSize.cs

```diff
@@ -253,21 +292,34 @@ public int CompareTo(ByteSize other)
             return this.Bits.CompareTo(other.Bits);
         }
 
+        /// <summary>
+        /// Returns a new ByteSize object whose value is the sum of the specified
+        /// value and this instance.
+        /// </summary>
+        /// <param name="bs">The ByteSize instance to sum.</param>
         public ByteSize Add(ByteSize bs)
         {
             return new ByteSize(this.Bytes + bs.Bytes);
         }
 
+        /// <inheritdoc cref="Add" />
+        /// <param name="value">The number of bits.</param>
         public ByteSize AddBits(long value)
         {
             return this + FromBits(value);
         }
 
+        /// <inheritdoc cref="Add" />
+        /// <param name="value">The number of bytes.</param>
         public ByteSize AddBytes(double value)
         {
             return this + ByteSize.FromBytes(value);
         }
 
+        /// <summary>
+        /// Subtracts the specified ByteSize object with this instance.
```

Comment:
The wording on `Subtract` is currently very different than on `Add`. I think it would be nice to be more consistent across the operations, either by making this a bit more verbose and similar to the `Add` docs, or making the `Add` docs simpler like this one.

Reply:
Minor: usually, you say you "add to" and "subtract from", so I'd suggest
```suggestion
        /// Subtracts the specified ByteSize object from this instance.
```